### ISSUE TITLE
[Accessibility][Share] 3rd party Pinterest share accessibility issues

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1803]
**Steps to Reproduce**
1. Press the TAB key to move focus to the Pinterest embed. Observe that it cannot receive keyboard focus or be activated with keyboard.
2. Inspect contrast of links in twitter embeds using a contrast tool such as the [Accessible Color Picker](https://chrome.google.com/webstore/detail/accessible-color-picker/bgfhbflmeekopanooidljpnmnljdihld?hl=en-US).

**Current Behavior**
* Pinterest embed in URL example cannot be operated with keyboard alone.

* Twitter embed in oEmbed URL example: links do not meet minimum contrast requirements (link color: #1b95e0, background color: #ffffff, contrast ratio 3.3:1; background on hover changes to #f7f9f9, contrast ratio 3.1:1), making them difficult to read for users with low vision.

**Expected behavior/code**
All content, including 3rd party content, should be accessible. Pinterest embed must be able to receive keyboard focus and be activated with keyboard alone. Twitter embed links must provide at least 4.5:1 contrast against the background.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Keyboard-only users, users with low vision.

**WCAG SC**
2.1.1 Keyboard
1.4.3 Contrast (Minimum)

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by gabrielwalt** [comment]
We'll be dropping the Pinterest share features in the new page version: see #1918.
